### PR TITLE
P: Fix https://github.com/AdguardTeam/AdguardFilters/issues/63887

### PR DESCRIPTION
**Pull request by Yuki2718**
https://github.com/AdguardTeam/AdguardFilters/issues/63887
You need to play the video for some time to reproduce. bann.js is not related to the issue but better to block this too.

### CODE REVIEW

File: easylist_adult/adult_specific_block.txt [hunk 1]
```diff
@@ -411,6 +411,7 @@
 ||jav-porn.net/js/popup.js
 ||javfor.me^*/banner/
 ||javhub.net/img/r.jpg
+||javporn.best/asset/bann.js
 ||javporn.in/clicunder.js
```
Comment (spirillen) [inline easylist_adult/adult_specific_block.txt:415]: This domain is hijacked by `parkingcrew.net.` and no longer serving adult material by google ads

```
javporn.in.     3600    IN      NS      ns2.parkingcrew.net.
javporn.in.     3600    IN      NS      ns1.parkingcrew.net.
```

<details><summary>Own note</summary>

https://www.mypdns.org/T635

</details>

Comment (Yuki2718) [reply easylist_adult/adult_specific_block.txt:415]: Hmm...I get  "Error 1020 Access denied".

Comment (spirillen) [reply easylist_adult/adult_specific_block.txt:415]: Yeah, parkingsuckers currently luckily having issues at the moment :smiley: :joy: 

The same happens with one if the other domain creepers but in reality [parkingcrew](https://www.mypdns.org/T635) as well

```shell
javstreaming.net.       172800  IN      NS      ns1.hastydns.com.
javstreaming.net.       172800  IN      NS      ns2.hastydns.com.
```

Comment (Yuki2718) [reply easylist_adult/adult_specific_block.txt:415]: Thx, I removed the rule from my list. Pingging @ryanbr in case not aware of.

**Pull request closed without merging**